download.sh

```diff
@@ -13,21 +13,37 @@ N_SHARD_DICT["30B"]="3"
 N_SHARD_DICT["65B"]="7"
 
 echo "Downloading tokenizer"
-wget ${PRESIGNED_URL/'*'/"tokenizer.model"} -O ${TARGET_FOLDER}"/tokenizer.model"
-wget ${PRESIGNED_URL/'*'/"tokenizer_checklist.chk"} -O ${TARGET_FOLDER}"/tokenizer_checklist.chk"
+mkdir -p "${TARGET_FOLDER}"
+CHK_FILE="tokenizer_checklist.chk"
+[ -f "${TARGET_FOLDER}/${CHK_FILE}" ] && MD5SUM_RESULT=$(cd "${TARGET_FOLDER}" && md5sum -c "${CHK_FILE}")
+if [ $? -ne 0 ]; then
+    wget --no-check-certificate ${PRESIGNED_URL/'*'/"${CHK_FILE}"} -O "${TARGET_FOLDER}/${CHK_FILE}"
```

Comment:
I would prefer certificate checking to continue be the default behavior.  However, if we wrapped the --no-check-certificate behind a ENV var or argument so that it can be selectively enabled, I would be interested in that.